bin/run-versioned-tests.sh

```diff
@@ -5,7 +5,7 @@
 
 set -x
 
-VERSIONED_MODE="${VERSIONED_MODE:---major}"
+VERSIONED_MODE="${VERSIONED_MODE:---minor}"
```

Comment:
revert this, i was just doing this to make sure my changes worked on all minor versions